src/style.css

```diff
@@ -0,0 +1,110 @@
+@font-face {
+  font-family: "Avenir", sans-serif;
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "AvenirHeavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
```

Comment:
Remove redundant default values. Fix it everywhere

Reply:
done

Reply:
font-style: normal; is a default property, remove it plz

Reply:
Not fixed 
![image](https://user-images.githubusercontent.com/97547824/230759989-1938ff10-55a1-46ae-84a8-f4729584c911.png)